packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/nodes/node/dashboard/dashboard.routing.js

```diff
@@ -241,12 +241,15 @@ export default /* @ngInject */ ($stateProvider) => {
             productId: serverName,
           },
         ),
-      orderPrivateBandwidthLink: /* @ngInject */ ($state, serverName) =>
+      getOrderPrivateBandwidthLink: /* @ngInject */ (
+        $state,
+        serverName,
+      ) => () =>
```

Comment:
Why do you need to create a function ? It should be OK with the code as is (with the property)

Reply:
Its not able to find the state at initial resolve hence modifying it to be as funtcion.